tests/DB/QueryBuilderTest.php

```diff
@@ -177,6 +191,20 @@ public function testOrderByMulti()
 		$this->assertEquals($expected, trim(preg_replace('/\s+/', ' ', $query)));
 	}
 
+	public function testOrderByArray()
+	{ 
+		$query = $this->_builder
+			->select("*")
+			->from('table')
+			->orderBy(['col', 'col2'])
+			->getQueryString()
+		;
+
+		$expected = "SELECT * FROM table ORDER BY col, col2";
+
+		$this->assertEquals($expected, trim(preg_replace('/\s+/', ' ', $query)));
+	}
+
 	public function testLimit()
```

Comment:
Do you think we should add some tests for calling `orderBy()` and `groupBy()` multiple times?